test/common/config/delta_subscription_test_harness.h

```diff
@@ -133,7 +147,7 @@ class DeltaSubscriptionTestHarness : public SubscriptionTestHarness {
     last_response_nonce_ = std::to_string(HashUtil::xxHash64(version));
     response->set_nonce(last_response_nonce_);
     response->set_system_version_info(version);
-    response->set_type_url(Config::TypeUrl::get().ClusterLoadAssignment);
+    response->set_type_url(type_url_);
```

Comment:
I can see the point of testing different transport versions in these tests, but I don't think we need to vary the resource version.

Reply:
only to use `SUPPORTED_MAJOR_API_VERSION` for now.